### ISSUE TITLE
Working with Windows

### ISSUE THREAD

**Issue by jududdar** [Sarjuuk/aowow#25]
It appears from the most recent commit that you're doing some work to make this compatible with Windows.  I've ran into a couple of hiccups while setting this up and thought I'd document them here:

1.  ~~After entering database config during first run, the walkthrough exits as soon as I press Enter.  This was easy to get around as I just had to type `php aowow --firstrun --resume`~~ After some tweaks, ran again and didn't have to ever resume the firstrun.  Disregard.

2.  Once I got that sorted out, the installer kept dying while importing Spell.dbc.  Went to task manager and noticed my PHP got to 128M and exited.  Changed memory_limit variable in php.ini a few times until I finally set it to 2048M which allowed me to continue.  (I think the highest the memory got was about 1280M)

3.  Some of the hardcoded time limits in the filegen files weren't long enough.  It could be my machine, but resources weren't out of hand while script was running.  Files below:
 * /setup/tools/filegen/enchants.func.php -> `set_time_limit(180)`
 * /setup/tools/filegen/profiler.func.php Mounts section -> `set_time_limit(250)` (this was too high I'm sure, tried to tweak it to a sane amount, but never could get it right)
 * /setup/tools/filegen/profiler.func.php Companions section -> `set_time_limit(50)`

After these changes, I've ran through the installer start to finish without anything hanging up.  I couldn't ever get the MPQ Extractor to compile properly under Win7 x64, so I used MPQEditor (from http://www.zezula.net/en/mpq/download.html) and imported all MPQs in merge mode, then cherry picked what was needed.  The site is working great!  I'm still exploring each area to make sure I got everything right.

No showstopping issues with your code at all, just sharing some guidance for us Windows folk!

**Comment by Sarjuuk** [comment]
1. ok  :o
2. yes, parsing the spell.dbc is ab bit crazy, i failed at making it run efficiently .. anyway
I'd advise against setting such variables globally. either edit it into the .htaccess manually or  add a new php-setting to the ?_config - table.
3. i guess, i'll have to recheck these values.

I'm glad to hear it worked o.k. though :)

**Comment by jududdar** [comment]
Mostly everything is working great!  Some friends and I used the site quite extensively while we played tonight.  I still can't get 3D views on items, and no quests work, but it appears that the quest tables might be in transition anyway.  I matched the fields in quests.func.php and letting the script do its thing again to see if I got it right.
I'm assuming the 3D won't load due to not having BLPConverter on Windows.  I can live without those, though.  I guess I could just move the server and site over to a Linux VM, but now it's becoming a fun challenge to tinker with.
